s2stac/src/main/scala/Main.scala

```diff
@@ -0,0 +1,24 @@
+package com.azavea.s2stac
+
+import cats.effect._
+
+import com.monovore.decline._
+import com.monovore.decline.effect._
+
+object HelloWorld
+    extends CommandIOApp(
+      name = "s2stac",
+      header = "Convert a Sentinel-2 inventory file into a STAC-compliant catalog"
+    ) {
+
+  override def main: Opts[IO[ExitCode]] =
+    Commands.createCatalogOpts
+      .map({ cmd =>
+        IO {
+          println(s"Output root: ${cmd.outputCatalogRoot.value}")
```

Comment:
One piece of feedback and really it's not important, more of a suggestion if you're interested in it.

I think it would be good if we start with better coloring/formatting of our log messages from the beginning and can normalize better output from the start. For instance, there's a library [fansi](https://github.com/lihaoyi/fansi) you could use to color the output green or something so let people know this is expected output. Also you could introduce some whimsy with [emojis](https://github.com/lightbend/lightbend-emoji).

This is a pretty nit-picky thing and also more of a personal preference that I'd like to see become more standard in our projects, but also I don't feel particularly strong here.

Reply:
It was pretty easy and I'm also making heavy use of debug commands (since additional commands are so cheap) so I went ahead and added it here -- this is info/debug/error/warn:

![image](https://user-images.githubusercontent.com/5702984/87953560-cfd1c300-ca70-11ea-9c10-7670300706c7.png)

Reply:
nice, those look 💯